apis/core/lib/domain/csv-mapping/delete.ts

```diff
@@ -0,0 +1,27 @@
+
+import { NamedNode } from 'rdf-js'
+import { ResourceStore } from '../../ResourceStore'
+import { getSourcesFromMapping } from '../queries/csv-source'
+
+import { deleteSourceWithoutSave } from '../csv-source/delete'
+import { cc } from '@cube-creator/core/namespace'
+
+export async function deleteMapping(csvMapping: NamedNode, store: ResourceStore): Promise<void> {
+  const sources = await getSourcesFromMapping(csvMapping)
+  for (const source of sources) {
+    await deleteSourceWithoutSave(source, store)
+  }
```

Comment:
But besides, why not using the store?

```suggestion
  const csvMappingResource = await store.get(csvMapping)
  const sources = csvMappingResource.out(cc.csvSource).terms
  for await (const source of sources) {
    if (source.termType === 'NamedNode') {
      await deleteSource({resource: source, store})
    }
  }
```

Reply:
Actually I did not need to load the graph at the beginning. But it is way cleaner without a query, in my opinion. I go with this one